packages/icons/src/index.js

```diff
@@ -136,6 +140,7 @@ export { default as plugins } from './library/plugins';
 export { default as plusCircleFilled } from './library/plus-circle-filled';
 export { default as plusCircle } from './library/plus-circle';
 export { default as plus } from './library/plus';
+export { default as postAuthor } from './library/post-author';
```

Comment:
I was curious why this icon was added, because the old post author icon was already in use in the Post Author block:

<img width="296" alt="Screenshot 2021-06-02 at 09 55 50" src="https://user-images.githubusercontent.com/1204802/120445797-445fd880-c389-11eb-9195-e73a8948986d.png">

As it turns out, we have _to_ copies of the old icon already in use, included with the post author block and post comment author blocks, respectively, rather than the icon from the library being used:

<img width="839" alt="Screenshot 2021-06-02 at 10 00 07" src="https://user-images.githubusercontent.com/1204802/120445951-68bbb500-c389-11eb-8aed-5da9a89649c3.png">

<img width="773" alt="Screenshot 2021-06-02 at 10 00 14" src="https://user-images.githubusercontent.com/1204802/120445954-69ece200-c389-11eb-955f-e77437e84ba4.png">

So those two icons need to be removed entirely, and the post author and post comment icons need to be updated to point to the new canonical Post Author icon you're here adding to the icon library. I can help with that if you like, just let me know!

Reply:
Done in https://github.com/WordPress/gutenberg/pull/32371/commits/8bc8166c63efaafdde67418b6c7ea5de5c2d165c — let me know if I missed anything.

Reply:
Oh that seems cool!